### ISSUE TITLE
Variances in caret icon sizes in terra-section-header and clinical-data-grid

### ISSUE THREAD

**Issue by medhapatil12** [cerner/terra-clinical#807]
# Bug Report

## Description
Noticed that the dynamic group label carets are a bigger size than the section carets. 
Ideally it should be the same size. 

As part of our analysis, the size of the icons differ because: 
The outer label is from Clinical trend view which uses terra-section-header which has an accordion-icon.
The inner section is a clinical-data-grid which has an icon named as IconCaretDown.

## Steps to Reproduce

1.  Open Vitals Viewer on Powerchart on 64Dev for the patient Gergo Polaristeam.
2. The dynamic group that is seen has different sized carets for the group label and the inner section.

## Additional Context / Screenshots

![image](https://user-images.githubusercontent.com/43102381/165309827-6d6c8767-7ffd-4888-b23c-554881de855a.png)

The carets for the Eating Difficulties and Incision wound are smaller in size than their inner sections.

## Expected Behavior
The carets must be the same size.

## Possible Solution
terra-section-header and clinical-data-grid to use the same caret icon. 

## Environment
* Component Name and Version: ion-vitals-js version [1.13.0]
* Browser Name and Version: Google Chrome version 100.0.4896.88
* Node/npm Version: Node 10.24.1/npm 6.14.12
* Webpack Version: 4.46.0
* Operating System and version (desktop or mobile): Mac OS version 12.3.1

**Comment by chrismichalewicz** [comment]
UXPLATFORM-6610 has been logged to track this internally
